evoko-build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Funderar på om det är bättre att ha den här filen i evoko-getaroom och kanske lägga in react-native som en git submodule? Eller är det dumt?